src/Test/DepTyCheck/Gen.idr

```diff
@@ -4,8 +4,6 @@ import Control.Monad.Maybe
 import public Control.Monad.Error.Interface
 import Control.Monad.Random
 import public Control.Monad.Random.Interface
-import Control.Monad.State
-import public Control.Monad.State.Interface
```

Comment:
`MonadState` is not used in this file

Reply:
Right, I checked, this was before introducing the `Control.Monad.Random.Interface` and previous design of the `random-pure` library's predecessor. I think, it's okay to remove this now.